com.ibm.wala.cast.js/source/com/ibm/wala/cast/js/loader/JavaScriptLoader.java

```diff
@@ -263,7 +262,7 @@ public SSAPutInstruction PutInstruction(int iindex, int ref, int value, String f
             byte[] utf8 = field.getBytes("UTF-8");
             return PutInstruction(iindex, ref, value, 
                 FieldReference.findOrCreate(JavaScriptTypes.Root, Atom.findOrCreate(utf8, 0, utf8.length), JavaScriptTypes.Root));
-          } catch (UnsupportedEncodingException e) {
+          } catch (@SuppressWarnings("unused") UnsupportedEncodingException e) {
```

Comment:
Yes, we can definitely turn off the Eclipse warning for unused exception parameters, and get rid of those per-location suppression declarations. Given that there are 157 such cases, that seems perfectly reasonable. I'll make that change and merge it into this existing pull request.